transformer_engine/pytorch/module/base.py

```diff
@@ -48,6 +48,7 @@
 _cublas_workspace = None
 _ub_communicators = None
 _NUM_MAX_UB_STREAMS = 3
+# Must be identical to `num_streams` in transformer_engine/common/include/transformer_engine/gemm.h
 _NUM_MAX_CUBLAS_STREAMS = 4
```

Comment:
Could you consider exposing this variable from CPP to Python via pybind to make sure it has a correct value?

Reply:
Changed accordingly.